spk/tor/src/wizard/install_uifile

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "step_title":"TOR Relay Type",
+        "items":[
+            {
+                "type":"singleselect",
+                "desc":"Please select your prefered TOR Relay Type",
+                "subitems":[
+                    {
+                        "key":"TOR_NONEXIT",
+                        "desc":"Non-exit (Configure TOR as an Non-exit Relay)",
```

Comment:
I'd propose to use `non-exit relay`, `exit relay` and `anonymous bridge`, to keep in line with Tor's usage of caps.